test/commands/addCommand.spec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var config = require('../lib/config');
+var helper = require('../helper');
+var redis = config.redis;
+var assert = require('assert');
+
+describe("The 'addCommand/add_command' method", function () {
+
+    helper.allTests(function (parser, ip, args) {
```

Comment:
You do not have to execute these tests for different kinds of options passed to the redis instance. This is only necessary in case the client might behave differently due to the different options (e.g. comparing the JS parser to the hiredis parser).